glossary.html

```diff
@@ -52,7 +52,7 @@ <h1>Acronym Glossary</h1>
                 </blockquote>
             <p id="page-copyright">&copy; 2019 <a href="mailto:Brett.Marshall@student.saultcollege.ca?subject=Messsage%20from%20CSD120%20Website">Brett Marshall</a></p>
         </footer>
-        <div class="backtotop">
+        <div class="back-to-top">
```

Comment:
incorrect class name (not a big deal, but I need these things to be correct to make grading much easier)